apps/web/src/components/Charts/Transaction/DailyAvgMaxBlobGasFeeChart.tsx

```diff
@@ -16,20 +18,21 @@ export type DailyAvgMaxBlobGasFeeChartProps = {
 export const DailyAvgMaxBlobGasFeeChart: FC<
   Partial<DailyAvgMaxBlobGasFeeChartProps>
 > = function ({ days, avgMaxBlobGasFees, compact = false }) {
+  const { converted, unit } = useArrayBestUnit(avgMaxBlobGasFees);
+
   const options: EChartOption<
     EChartOption.SeriesBar | EChartOption.SeriesLine
   > = {
     ...buildTimeSeriesOptions({
       dates: days,
       axisFormatters: {
-        yAxisTooltip: (value) => formatWei(value),
-        yAxisLabel: (value) => formatWei(value),
```

Comment:
Is it possible to keep yAxisLabel with the units?

![image](https://github.com/user-attachments/assets/1505e1cd-5a2a-402f-ae0b-69789ed3bccb)